support-frontend/assets/components/directDebit/directDebitProgressiveDisclosure/components/playback.jsx

```diff
@@ -18,55 +53,54 @@ function Playback(props: {
+    <div css={directDebitForm}>
+      <label htmlFor="account-holder-name-input" css={fieldLabel}>
         Account name
       </label>
-      <span>
+      <span css={fieldData}>
         {props.accountHolderName}
       </span>
 
-      <label htmlFor="sort-code-input" className="component-direct-debit-form__field-label">
+      <label htmlFor="sort-code-input" css={fieldLabel}>
         Sort Code
       </label>
-      <span>
+      <span css={fieldData}>
         {props.sortCodeString}
       </span>
 
-      <label htmlFor="account-number-input" className="component-direct-debit-form__field-label">
+      <label htmlFor="account-number-input" css={fieldLabel}>
         Account number
       </label>
-      <span>
+      <span css={fieldData}>
         {props.accountNumber}
       </span>
 
-      <label htmlFor="confirmation-text__locked" className="component-direct-debit-form__field-label">
+      <label htmlFor="confirmation-text__locked" css={fieldLabel}>
         Declaration
       </label>
-      <div id="confirmation-text__locked" className="component-direct-debit-form__confirmation-text__locked">
+      <p id="confirmation-text__locked" css={fieldInfo}>
         I have confirmed that I am the account holder and that I am solely able to authorise debit
         from the account
-      </div>
-      <div className="component-direct-debit-form__confirmation-guidance">
-        If the details above are correct press confirm to set up your direct debit, otherwise press
+      </p>
+      <p css={fieldInfoWithMargin}>
+        If the details above are correct, press confirm to set up your direct debit, otherwise press
         back to make changes
-      </div>
+      </p>
 
-      <div className="component-direct-debit-form__cta-container">
-        <button
-          className="component-direct-debit-form__cta component-direct-debit-form__cta--edit-button focus-target"
-          onClick={props.editDirectDebitClicked}
-        >
-          <span className="component-direct-debit-form__cta-text component-direct-debit-form__cta-text--inverse">Edit</span>
-        </button>
-        <button
-          id="qa-submit-button-2"
-          className="component-direct-debit-form__cta component-direct-debit-form__cta--confirm-button focus-target"
-          onClick={e => props.onSubmit(e)}
-        >
-          <span className="component-direct-debit-form__cta-text">{props.buttonText}</span>
-          <div className="component-direct-debit-form__arrow"><SvgArrowRightStraight /></div>
-        </button>
+      <div css={ctaContainer}>
+        <ThemeProvider theme={buttonReaderRevenueBrandAlt}>
+          <Button onClick={props.editDirectDebitClicked}>Edit</Button>
+        </ThemeProvider>
+        <ThemeProvider theme={buttonReaderRevenueBrand}>
+          <Button
+            id="qa-submit-button-2"
+            onClick={e => props.onSubmit(e)}
```

Comment:
Very minor suggestion, could we just state `onClick={props.onSubmit}` as we do for the 'Edit' button rather than using an arrow function? For consistency.

Reply:
In this case, onSubmit is expecting to receive an event (for preventDefault) whereas the edit button's click handler isn't expecting any arguments. So the format of each is consistent with its function call. If consistency is important, perhaps both should be arrow functions?

Reply:
I believe if you used `onClick={props.onSubmit}` then the `onSubmit` function would still receive the `Event` as an argument, I can see that this arrow function is more explicit about what it's being passed to it for someone reading the code though, I'd be happy for it to stay as is.

Reply:
Okay, sure.

Reply:
And then after further thought and research I have taken it out! Thanks for drawing my attention to it George.